AnimalsTest/IBigCats.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AnimalsTest
+{
+    public interface IBigCats
+    {
+       bool IsValidName(string name);
+        /// <summary>
+        /// returns animal voice
+        /// </summary>
+        /// <returns></returns>
+        string Voice();
+        /// <summary>
+        /// returns distance which animal will be able to run for a certain time
+        /// </summary>
+        /// <param name="time"></param>
+        /// <returns></returns>
+        int Distance(int hours);
```

Comment:
```suggestion
        bool IsValidName(string name);
        
        /// <summary>
        /// Returns animal voice.
        /// </summary>
        string Voice();
        
        /// <summary>
        /// Returns distance which animal is able to run for a certain time.
        /// </summary>
        int Distance(int hours);
```

In this chunk there are some formatting errors, consider enabling editorconfig rules to detect them.
Also if certain doc is not filled it is better not to keep it.
StyleCop analyzer has a lot of useful rules for documentation: 16xx range https://documentation.help/StyleCop/Documentation%20Rules.html

imo SA1603,  SA1625,  SA1626,  SA1628 - 1630, SA1644 are helpful

Reply:
Formatting issues are in all the files.